Makefile

```diff
@@ -252,11 +252,13 @@ sdist-venv: clean
 	rm -rf dist venv
```

Comment:
I wasn't sure if this target is still being used anywhere -- so I kind of hacked it to install the dev dependencies to get it to pass on my local